Graphs/utils/DataParser/index.js

```diff
@@ -3,8 +3,9 @@ import stackedParser from './stackedParser';
 import multiLineParser from './multiLineParser';
 
 export default (props) => {
+    const multiLineGraph = Array.isArray(props.key) || props.graph === 'MultiLineGraph';
```

Comment:
avoid hardcoding, use GraphTypeEnum.MultiLineGraph.name

Reply:
@killanch 

`GraphTypeEnum.MultiLineGraph.name` we can not  use this. because it is in the `vsd-react-ui`, And `vis-graph` is an independent submodule it is also used in `visualization-framework`.